ios/Gemfile.lock

```diff
@@ -70,24 +70,27 @@ GEM
     nap (1.1.0)
     netrc (0.11.0)
     public_suffix (4.0.6)
+    rexml (3.2.5)
     ruby-macho (1.4.0)
     thread_safe (0.3.6)
     typhoeus (1.4.0)
       ethon (>= 0.9.0)
     tzinfo (1.2.9)
       thread_safe (~> 0.1)
-    xcodeproj (1.19.0)
+    xcodeproj (1.20.0)
       CFPropertyList (>= 2.3.3, < 4.0)
       atomos (~> 0.1.3)
       claide (>= 1.0.2, < 2.0)
       colored2 (~> 3.1)
       nanaimo (~> 0.3.0)
+      rexml (~> 3.2.4)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
   cocoapods (= 1.10.1)
+  ethon (>= 0.13.0)
 
 BUNDLED WITH
-   2.1.4
+   2.2.22
```

Comment:
in the docs you talk about using bundler 2.1.4 which was previously used but here I see bundler 2.2.22, is that intended and should we bump the version everywhere ?

Reply:
this was probably bumped when asked to upgrade some other files, 0/5